src/content/docs/tutorial-service-level-mgmt/intro-to-slm.mdx

```diff
@@ -0,0 +1,156 @@
+    * **Customer-facing service boundary**: The customer-facing GraphQL API. This boundary gives you the total sum response time and success metrics required for measuring performance health, without the need to measure each dependency.
+  </Collapser>
+
+  <Collapser
+    id="slm-terminology-error-budget"
+    title="Error budget"
+  >
+An error budget measures and communicates your service level objective compliance. We have more detailed information on this advanced method in our [service level management docs](/docs/service-level-management/intro-slm).
+
+An error budget tracks how much your service can fail before becoming "out of compliance." You can use a few different methods for defining and measuring error budgets. The word "error" in "error budget" doesn't mean HTTP request errors or application errors, but instead refers to "any bad request or event" as defined in your service level objectives.
+
+**Note**: We recommend you first learn how to define, calculate, and communicate your service level compliance before implementing error budgets. It's very important that you can establish a target for you SLIs to measure the success of how well your SLI meets your targets. For example, you can track whether or not you're within 500 milliseconds of and error-free on 99.99% each for a 24-hour period of your login requests. If your 24-hour SLI compliance result meets 99.99%, then you can consider your objective met.
+
+**References**
+
+Google defines an error budget as:
+
+"In a nutshell, an error budget is the amount of error that your service can accumulate over a certain period of time before your users start being unhappy. Once you define an objective for each (SLI) for your service-level objective (SLO) — the error budget is the remainder, up to 100." -- [Google Cloud - How maintenance windows affect your error budget—SRE tips](https://cloud.google.com/blog/products/management-tools/sre-error-budgets-and-maintenance-windows)
+
+Atlassian defines an error budget as:
+
+"An error budget is the maximum amount of time that a technical system can fail without contractual consequences. For example, if your service level agreement (SLA) specifies that systems will function 99.99% of the time before the business has to compensate customers for the outage, that means your error budget (or the time your systems can go down without consequences) is 52 minutes and 35 seconds per year." -- [Atlassian - What is an error budget and why does it matter?](https://www.atlassian.com/incident-management/kpis/error-budget)
+  </Collapser>
+
+## Prerequisites [#prerequisites]
+
+* [APM instrumentation](/docs/apm/new-relic-apm/getting-started/introduction-apm) of your primary application.
```

Comment:
I might have a couple sentences here that explains why these are prerequisites. For example, if you aren't already monitoring an entity, then this kind of "Advanced" monitoring practice is moot. And I'd also mention that synthetics are easy ways to generate metrics w/o relying on customers. Your service lvls are only as good as the amount of data that's informing your decisions

Reply:
I've fleshed this out a little bit more and helped it to flow a bit better as well. Thanks for the call out!